ekino/newrelic-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Ekino\\NewRelicBundle\\EkinoNewRelicBundle": ["all"]
```

Comment:
What's about `["prod"]` and moving `packages/ekino_new_relic.yaml` to `packages/prod/ekino_new_relic.yaml`?

Reply:
The 1.3 recipe did that. I chose not to for several reasons: 
- If you have invalid config you will only notice that in production
- You cannot write code that interact with services provided by the bundle. 
- We got a great BlackholeInteractor that we should use as often as we can. =)